solidity/contracts/bridge/Bridge.sol

```diff
@@ -178,6 +178,8 @@ contract Bridge is Governable, EcdsaWalletOwner {
     event MovingFundsParametersUpdated(
         uint64 movingFundsTxMaxTotalFee,
         uint32 movingFundsTimeout,
+        uint96 movingFundsTimeoutSlashingAmount,
+        uint256 movingFundsTimeoutNotifierRewardMultiplier,
         uint64 movingFundsDustThreshold
     );
```

Comment:
In a separate PR, can we align the order of parameters in `*ParametersUpdated` events, `update*`, and `*Parameters()` functions so that timeout, slashing amount, and notifier reward multiplier would be always at the end? 

`RedemptionParametersUpdated` would stay as-is.

`MovingFundsParametersUpdated` would become:
```
event MovingFundsParametersUpdated(
    uint64 movingFundsTxMaxTotalFee,
    uint64 movingFundsDustThreshold,
    uint32 movingFundsTimeout,
    uint96 movingFundsTimeoutSlashingAmount,
    uint256 movingFundsTimeoutNotifierRewardMultiplier
);
```

`FraudParametersUpdated` would become:
```
event FraudParametersUpdated(
  uint256 fraudChallengeDepositAmount,
  uint256 fraudChallengeDefeatTimeout,
  uint96 fraudSlashingAmount,
  uint256 fraudNotifierRewardMultiplier
);
```

Reply:
Done in 0c319131bd48522f669c1148f7f512b6098c6222.